assets/javascripts/discourse/components/ai-persona-llm-selector.gjs

```diff
@@ -0,0 +1,181 @@
+import Component from "@glimmer/component";
```

Comment:
what are the big change sin this file?

Reply:
Nothing just copy/pasted into its own component?

Reply:
Simply abstracting out the "setting" of values -- they're passed in now. instead of `this.composer.metaData["personaId"] = id` we have `this.args.setPersonaId(id)`